ref/commands/server/server-windowsservice.adoc

```diff
@@ -6,74 +6,88 @@
 //
 // Contributors:
 //     IBM Corporation
+:page-layout: server-command
+:page-type: command
+= server WinService command
 
-= server start command
+Register, unregister, start, and stop the named Open Liberty server as a Windows service.
 
-Starts the named Open Liberty server as a Windows service (Windows only).
+The Windows service name and Windows service display name is `serverName`. After you register, you can find and administer the Open Liberty server as a service by using `regedit` from the Windows Registry program. Entries for the service in the registry are in the following locations:
```

Comment:
I don't think this makes sense where it is. In the original documentation this text was in the context of registerWinService where it makes sense, since it tells you how you can find the windows service in the windows service control panel view, but it doesn't really apply to the start/stop or unregister option.